.changeset/silver-lies-rest.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-catalog-backend-module-gitlab': minor
```

Comment:
```suggestion
'@backstage/plugin-catalog-backend-module-gitlab': patch
```

package is 0.x so we can add deprecations in `patch` releases. The future breaking change will however need to be a `minor` release :)